Documentation/CRDs/Cluster/external-cluster/topology-for-external-mode.md

```diff
@@ -0,0 +1,199 @@
+# Topology provisioning for external mode
+
```

Comment:
For this whole doc, think about what the user needs to do for accomplishing the configuration of a topology-based storage class. Only include that in the doc. Remove all extra output that distracts from these steps. Try to make this doc concise, and not show all possible output.

Reply:
okay